src/pages/MenuPage.js

```diff
@@ -136,8 +136,8 @@ export class MenuPage extends React.Component {
         key={page}
         style={menuButton}
         textStyle={menuButtonText}
-        text={title}
-        onPress={() => navigateTo(page, title)}
+        text={navStrings[title]}
```

Comment:
Ah, of course.

I think this means we can remove the database `eventlistener` in `componentDidMount` - (this callback seems to be missing, anyway)

And that also means from `componentWillUnmount`